app/models/config_report.rb

```diff
@@ -82,9 +82,12 @@ def summary_status
   end
 
   delegate :error?, :changes?, :pending?, :status, :status_of, :to => :calculator
+  # delegate all metric methods
   delegate(*METRIC, :to => :calculator)
+  # delegate all metric_s (as string) methods
+  delegate(*(METRIC.map { |x| "#{x}_s".to_sym }), :to => :calculator)
```

Comment:
Methods `applied_s` return number as string: `"0"` and if that's max value then it returns it with plus `"1023+"`

Reply:
shouldn't we actually use the delegation to `status_text_of(<metric>)`? Probably something like:
```ruby
METRIC.each |metric|
  define_method("#{metric}_s")
    calculator.status_as_text_of(metric)
  end
end
```

Reply:
Since OpenSCAP plugin overrides the calculator in its own `ArfReportStatusCalculator` class I wanted to keep the current design of having those methods defined both on ConfigReport and StatusCalculator.